engine/src/card.cpp

```diff
@@ -1088,30 +1088,43 @@ void MCCard::timer(MCNameRef mptr, MCParameter *params)
 		MCObject::timer(mptr, params);
 }
 
+bool MCCard::isdeletable()
+{
+    bool t_deletable = true;
+    
+    if (t_deletable)
+        t_deletable = MCObject::isdeletable();
+    
+    if (t_deletable)
+        t_deletable = !getflag(F_C_CANT_DELETE);
```

Comment:
I'm thinking this may very well be a significant change of behavior we might or might not want to make where you can't delete a stack if it contains a group, card or substack (or any of the substack's groups and cards) with cantDelete true.

This was previously unchecked but I think it conforms to the docs:

> If an object's cantDelete property is set to true, the object cannot be deleted either by user action or by a handler. If you want to delete the object, you must first set its cantDelete to false.

I'm wondering if/how we can identify the specific object causing the error in this case as it's not necessarily in the trace like scriptdepth identifies. Ideally an error message would say "cantDelete of object <long name> true". I also need to check how this stuff works with revert and anything the IDE does to ensure a revert works as a result of user actions like "Close and remove from memory"... like setting the cantDelete of the mainstack but not substacks, cards or groups...